app/assets/javascripts/datepickers.js

```diff
@@ -47,6 +47,32 @@ function load_datepicker() {
       $('.date_end_no_min').datepicker( "option" , "minDate" , start_date);
     }
   });
-};
 
+  $('.date_checkout').datetimepicker({
+    dateFormat: "mm/dd/yy",
+    timeFormat: "h:mm TT",
+    altFieldTimeOnly: false,
+    altFormat: "yy-mm-dd",
+    altTimeFormat: "HH:mm",
+    altField: '#date_checkout_alt',
+    minDate: 0,
+    onClose: function(dateText, inst) {
+      var chaeckin_date = $('.date_checkin').datepicker("getDate");
+      var checkout_date = $('.date_checkout').datepicker("getDate");
+      if (checkout_date > chaeckin_date){
```

Comment:
I'm not thrilled with this for two reasons:

1. Setting the checkin date to one day after the checkout date seems arbitrary and dangerous - I feel like it should reset the field to the original value (if it's possible to store it).
2. The fact that it only runs on the checkout field doesn't make sense - what if you edit the checkin date to be earlier than the checkout date?

I feel like we should handle this with an ActiveRecord validation instead and allow people to edit the fields to whatever values they'd like, or simply revert changes that violate that validation if possible. What do you think?